cpu/cc26xx_cc13xx/power_arch.c

```diff
@@ -96,18 +96,24 @@ void power_enable_domain(const power_domain_t domain)
 void power_clock_enable_gpio(void)
 {
     PRCM->GPIOCLKGR = GPIOCLKGR_CLK_EN;
+    PRCM->GPIOCLKGS = GPIOCLKGS_CLK_EN;
+    PRCM->GPIOCLKGDS = GPIOCLKGDS_CLK_EN;
 
     prcm_commit();
 }
 
 void power_clock_enable_gpt(uint32_t tim)
 {
     PRCM->GPTCLKGR |= (1 << tim);
+    PRCM->GPTCLKGS |= (1 << tim);
+    PRCM->GPTCLKGDS |= (1 << tim);
```

Comment:
Probably worth amending the timer config struct so that timers can be flagged to keep running in deep sleep explicitly.

Reply:
I removed the deep sleep clock gating as it might not be desirable right now (we don't enter deep sleep). On a further PR I'll implement `pm_layered` properly.